src/ui-shell/ui-shell.module.ts

```diff
@@ -52,10 +58,12 @@ export {
 		HeaderGlobal,
 		HeaderAction,
 
+		Hamburger,
+
```

Comment:
One last thing, it looks like a few extra newlines sneaked in here (seems to be every instance of `Hamburger`)

Reply:
Sounds good. Made the changes, let me know if that's what you were after!